apps/k8s_utils.go

```diff
@@ -52,3 +52,47 @@ func CreateStorageClass(mb *mongoBuilder) (string, error) {
 	}
 	return name, nil
 }
+
+func CreatePostgresStorageClass(pb *postgresBuilder) (string, error) {
+	var err error
+	var poolsInCluster []common.MayastorPool
+	const sleepTime = 3
+
+	if pb.replicaCount == 0 && pb.architecture == Replication {
+		for ix := 0; ix < (k8stest.DefTimeoutSecs+sleepTime-1)/sleepTime; ix++ {
+			poolsInCluster, err = k8stest.ListMsPools()
```

Comment:
Don't use k8stest.ListMsPools() here otherwise it will be specific to mayastor. Going forward this will be used for local engine as well. Replica logic should be done at test label .

Reply:
This logic will be probably removed